src/oc_erchef/apps/chef_objects/src/chef_s3_ops.erl

```diff
@@ -137,6 +141,11 @@ check_file(OrgId, AwsConfig, Bucket, Checksum) ->
                      error_logger:error_msg("Checking presence of file (checksum: ~p) for org ~p from bucket ~p (key: ~p) failed because the file was not found~n",
                                             [Checksum, OrgId, Bucket, Key]),
                      {missing, Checksum};
+                 %% TODO(ssd) 2020-09-03: should likely remove this case when we move to erlcloud
+                 error:{aws_error, {socket_error,retry_later}} ->
+                     error_logger:error_msg("Checking presence of file (checksum: ~p) for org ~p from bucket ~p (key: ~p) returned retry_later (retries left: ~p)~n",
+                                            [Checksum, OrgId, Bucket, Key, AttemptsLeft - 1]),
+                     check_file(OrgId, AwsConfig, Bucket, Checksum, AttemptsLeft -1);
```

Comment:
[nit] This is a tiny nit, but `AttemptsLeft - 1` reads better than `AttemptsLeft -1`, doesn't it? 🤔